mixins/cross-compile.mixin

```diff
@@ -0,0 +1,50 @@
+{
+    "build": {
+        "armhf-generic_linux": {
+            "cmake-args": [
+                "-DCMAKE_SYSTEM_NAME=Linux",
```

Comment:
There is some overhead between the mixin and armhf-linux.cmake we should resolve.

Reply:
Fixed in PR #1